gpgme.go

```diff
@@ -389,7 +390,11 @@ func (c *Context) GetKey(fingerprint string, secret bool) (*Key, error) {
 	key := newKey()
 	cfpr := C.CString(fingerprint)
 	defer C.free(unsafe.Pointer(cfpr))
-	return key, handleError(C.gpgme_get_key(c.ctx, cfpr, &key.k, cbool(secret)))
+	err := handleError(C.gpgme_get_key(c.ctx, cfpr, &key.k, cbool(secret)))
+	if key.k == nil {
```

Comment:
Or... It may be an issue with the C library. And maybe gpgme should be fixed to error out when the key is not found. Either way, this solution is faster for us and doesn't require us to patch the C library.